opt_out_http_api/store/tests/test_memory.py

```diff
@@ -0,0 +1,14 @@
+"""
+Tests for IOptOutStore.
+"""
+from opt_out_http_api.store.memory import OptOutMemory
+from twisted.trial.unittest import TestCase
+from twisted.web.server import Site
+from twisted.internet import reactor
+from twisted.internet.defer import inlineCallbacks
+
+
+class TestMemory(TestCase):
+    def test_setup(self):
+        store = OptOutMemory()
+        assert store is not None
```

Comment:
This is a good first test. We should assert that ``store`` implements ``IOptOutMemory``, using, e.g.
```
from zope.interface.verify import verifyObject
...
def test_setup(self):
     ...
     verifyObject(store, IOptOutStore)
```